frontend/pipeline.yml

```
@@ -202,21 +214,21 @@ build:frontend:docker:
   artifacts:
     expire_in: 2w
     paths:
-      - frontend/coverage
-      - frontend/screenshots
-      - frontend/logs
-      - frontend/junit
+      - ${COVERAGE_DIR}
+      - ${SCREENSHOTS_DIR}
+      - ${LOGS_DIR}
+      - ${JUNIT_DIR}
     reports:
       junit:
-        - frontend/junit/results.xml
+        - ${JUNIT_DIR}/results.xml
     when: always
   tags:
     - hetzner-amd-ax42
 
 test:frontend:acceptance:
   extends: .template:test:frontend:acceptance
   script:
-    - GUI_REPOSITORY=$(pwd)/frontend SERVER_ROOT=$(pwd) ./frontend/tests/e2e_tests/run
+    - GUI_REPOSITORY=${FRONTEND_DIR} SERVER_ROOT=${CI_PROJECT_DIR} ${FRONTEND_DIR}/tests/e2e_tests/run
```

I think the script itself actually defines these variables as long as it's executed within the git repository.
```suggestion
    - tests/e2e_tests/run
```

---

It does, I missed that, thx. But looking at it now, they are default/fallback values:

```bash
# frontend/tests/e2e_tests/run
export SERVER_ROOT=${SERVIER_ROOT:-$(git rev-parse --show-toplevel)}  # <-- I'll fix the typo in another commit
export GUI_REPOSITORY=${GUI_REPOSITORY:-${SERVER_ROOT}/frontend}
export COMPOSE_FILE="${SERVER_ROOT}/docker-compose.yml:${GUI_REPOSITORY}/tests/e2e_tests/docker-compose.e2e-tests.yml"
```

I will remove setting the vars again from the CI, since they will be the same like you said. But I guess we do want to keep the option to override the ones in the `run` script for local running?

---

done